src/main/java/MessageParser.java

```diff
@@ -0,0 +1,20 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class MessageParser {
+    private static final String MESSAGE_HEADER_REGEX = "//(.)\n(.*)";
+    private Pattern messageHeaderPattern;
+
+    public MessageParser() {
+        messageHeaderPattern = Pattern.compile(MESSAGE_HEADER_REGEX);
+    }
+
+    public Message parse(String message) {
+        Matcher matcher = messageHeaderPattern.matcher(message);
+        if (matcher.find()) {
+            return new Message(matcher.group(1), matcher.group(2));
+        }
+        return new Message(null, message);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return new Message(DEFAULT_DELIMITERS, message);
```